functional-test/src/test/groovy/org/rundeck/util/gui/pages/execution/ExecutionShowPage.groovy

```diff
@@ -105,6 +107,11 @@ class ExecutionShowPage extends BasePage  implements ActivityListTrait{
         el optionValueSelected
     }
 
+    static WebElement getActivityExecStatusIcon(WebElement row) {
```

Comment:
This static method is not safe and hard to reuse.
It takes a generic WebElement and returns a generic WebElement.
What stops someone from passing in a WebElement that has nothing to do with the row?